### PR TITLE
Odometer in reports

### PR DESCRIPTION
**Pull request by ckrey**
Implements Odometer in Summary, Trips, and Stops Reports #652 
relies on corresponding PR in traccar/traccar

### CODE REVIEW

File: web/l10n/en.json [hunk 1]
```diff
@@ -410,4 +412,4 @@
     "categoryTractor": "Tractor",
     "categoryTruck": "Truck",
     "categoryVan": "Van"
-}
```
Comment (tananaev) [inline web/l10n/en.json]: Please don't change end line.

File: web/app/view/ReportController.js [hunk 2]
```diff
@@ -595,6 +611,10 @@ Ext.define('Traccar.view.ReportController', {
         dataIndex: 'startTime',
         xtype: 'datecolumn',
         renderer: Traccar.AttributeFormatter.getFormatter('startTime')
+    }, {
+        text: Strings.reportStartOdometer,
```
Comment (tananaev) [inline web/app/view/ReportController.js]: Same question here. Shouldn't we just call it odometer?

**Comment by ckrey** [comment]
ok, both done

File: web/l10n/en.json [hunk 3]
```diff
@@ -385,6 +385,7 @@
     "reportSpentFuel": "Spent Fuel",
     "reportStartOdometer": "Odometer Start",
     "reportEndOdometer": "Odometer End",
+    "reportOdometer": "Odometer",
```
Comment (tananaev) [inline web/l10n/en.json]: We already have odometer string.

**Comment by ckrey** [comment]
replace by PR #706

**Pull request closed without merging**